Where_Is_My_Wife/Assets/_Main/Scripts/Input/InputSources.cs

```diff
@@ -15,6 +15,19 @@ public interface IPlayerInputEvent
         Action LookUpAction { get; set; }
         Action<bool> LookDownAction { get; set; }
     }
+
+    public interface ISpecialInputEvent
```

Comment:
Why special and not part of the UI?

Reply:
7307ce77c324c327dc42980235a42a5c63fb742b